packages/react/src/ActionList/ActionList.stress.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+import type {Meta} from '@storybook/react'
+import type {ComponentProps} from '../utils/types'
+import {StressTest} from '../utils/StressTest'
+import {TableIcon} from '@primer/octicons-react'
+import {ActionList} from '.'
+
+export default {
+  title: 'StressTests/Components/ActionList',
```

Comment:
is there a way to make the StressTests folder not appear in production? deployments similar to the Dev stories? or is there a reason we'd want that in production?

Reply:
Oh let me take a look!

I think they are fine either way though. I always thought transparency is a good policy there and if I were a developer looking into the design system I'd appreciate having them.